TRADEMARK-COMMITTEE.md

```diff
@@ -78,7 +78,7 @@ the trademark committee invites your feedback there. See
 | Member                   | Organization | Profile                                |
 | ------------------------ | ------------ | -------------------------------------- |
 | Ron Avnur                | Google       | [@ronavn](https://github.com/ronavn)   |
-| Unknown - to be selected | IBM/RedHat   | Unknown                                |
+| Doug Davis               | IBM/RedHat   | [@duglin](https://github.com/duglin)   |
```

Comment:
Is there a collective name for the two?

I propose 'Barney' since blue + red = purple

Reply:
Given the recent announcement of NewCo, perhaps OldCo ?